radio/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Yeah okay I low key have no idea how to structure a bindgen project. There will obviously be other top level shared artifacts (robot models, protocol models, etc). Do you have opinions because I don't think I have super strong ones rn.

Reply:
I dunno the typical way to structure a bindgen project but generally for a folder hierarchy I'd make the subfolders contain stuff that is relevant to only that part of the hierarchy. So I'd probably keep the shared headers outside of the rust folder cause they're not just used by rust, and I'd keep the radio python files in the top level too.

It's not hard to move stuff around later but if we're going to be adding more stuff we could have:
* `common-packets`
  * `include`
    * c headers
  * `rust-lib`
    * rust specific files
* `radio-protocol`
  * python files
* whatever other shared models/artifacts

Don't have a strong opinion because it doesn't make a functional difference, just whatever's easiest to keep organized.

Reply:
so should the structure be

- common/
  - packets
    - include
    - ateam-common-packets
    - Cargo.toml